Gemfile

```diff
@@ -91,6 +91,9 @@ gem 'kaminari'
 # used to decode html in some imported strings
 gem 'htmlentities'
 
+# consume APIs
+gem 'httparty'
```

Comment:
@simaob do we need this? can't we just use the `Net` module?

Reply:
I love this gem, let me use it. I love to party!